### PR TITLE
ZER 38 Investigate and fix Rubocop verification issue

### PR DESCRIPTION
**Pull request by Har4enkoO**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by haliapats — DISMISSED** [review]

**Review by maksymst — APPROVED** [review]

File: .rubocop.yml [hunk 1]
```diff
@@ -5,10 +5,17 @@ AllCops:
...
     - 'tmp/**/*'
     - 'storage/**/*'
     - 'log/**/*'
+    - 'spec/**/*'
+    - 'test/**/*'
+    - 'node_modules/**/*'
+    - 'config/**/*'
+    - 'bin/**/*'
+    - 'Rakefile'
+    - 'Gemfile'
```
Comment (maksymst) [inline .rubocop.yml:18]: remove this

File: app/models/field.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 class Field < ApplicationRecord
   belongs_to :calculator
-  before_create :set_selector
+  before_create :manage_selector
```
Comment (maksymst) [inline app/models/field.rb:5]: `manage_selector ` method does not exist

**Review by maksymst — CHANGES_REQUESTED** [review]

File: .rubocop.yml (hunk 1 above)
Comment (Har4enkoO) [reply .rubocop.yml:18]: Thanks. Done.

File: app/models/field.rb (hunk 2 above)
Comment (Har4enkoO) [reply app/models/field.rb:5]: Thanks. Done

**Review by maksymst — APPROVED** [review]

File: spec/models/field_spec.rb [hunk 3]
```diff
@@ -1,25 +1,34 @@
...
     it {
       is_expected.to define_enum_for(:kind)
         .with_values(%i[form parameter result])
     }
   end
+end
+
+RSpec.describe Field, type: :model do
+  subject(:field) { build(:field) }
+
```
[the remark this answers is not in the document]
Comment (Har4enkoO) [reply spec/models/field_spec.rb]: Thanks. Changed 30th line.

**Review by maksymst — APPROVED** [review]

**Review by haliapats — APPROVED** [review]

**Pull request merged**